### PR TITLE
Applicant Timeline - Experience Tutorial step

### LINKED ISSUE #2993: Task - Applicant Timeline - Experience Tutorial step
# Description
A step that goes before the Experience timeline that has no interactions, simply explains things with static content.

# Images
https://xd.adobe.com/view/4ac43ee7-a919-4397-6888-ce98019603f7-3cf5/screen/79017fa7-923b-4915-8c65-afc088b56a84/Step-2B-Requirements-22

### PR DESCRIPTION
**Pull request by yonikid15**
Resolves #2993 

### Notes:
- [ ] Still needs translations.

### CODE REVIEW

File: resources/assets/js/components/Checkbox.tsx [hunk 1]
```diff
@@ -34,37 +26,22 @@ const Checkbox: React.FunctionComponent<CheckboxProps> = ({
...
-          checked={checked}
-          value={value}
-          onChange={onChange}
-          onBlur={onBlur}
-          required={required}
-        />
-        <span>{label}</span>
-      </label>
-    </div>
-    <span>{errorText || <FormattedMessage {...inputMessages.error} />}</span>
```
Comment (tristan-orourke) [inline resources/assets/js/components/Checkbox.tsx:67]: Don't we need still the error and required messages?

File: resources/assets/js/components/JobBuilder/Details/JobDetails.tsx [hunk 2]
```diff
@@ -573,16 +572,9 @@ export const JobDetails: React.FunctionComponent<JobDetailsProps> = ({
                       <div>
                         <ContextBlockItem
                           wrapperMargin="bottom(normal)"
-                          subtext={textToParagraphs(
```
Comment (tristan-orourke) [inline resources/assets/js/components/JobBuilder/Details/JobDetails.tsx:576]: Do we not want to use textToParagraphs here anymore?

File: resources/assets/js/getpdf.js [hunk 3]
```diff
@@ -3,17 +3,17 @@
 /* eslint-disable new-cap */
 /* eslint-disable no-plusplus */
 /* eslint-disable prefer-template */
-
 // run on Review Applicants page to export all applications to PDF
+// add bookmark to Chrome with this as the URL:
 
-// add bookmark to Chrome with this as the URL: javascript:function clickAllApplicants(){let t=document.querySelectorAll("div.box.lg-2of11.applicant-links > a:nth-child(1)");!function(t){(t=Array.prototype.slice.call(t)).map(function(t){return t.getAttribute("href")})}(t);for(var e=0;e<t.length;e++){var l=window.open(t[e],"_blank"),n=l.document.createElement("script");n.setAttribute("src",t[e].origin+"/js/getpdf.js"),l.document.body.appendChild(n)}}clickAllApplicants();
```
Comment (tristan-orourke) [inline resources/assets/js/getpdf.js:9]: I don't think the PR should be modifying this...

**Comment by yonikid15** [comment]
Making a new PR.

**Pull request closed without merging**